### ISSUE TITLE
Buttons all showing up with same device name - not individual button names

### ISSUE THREAD

**Issue by jfinestone** [senorshaun/homebridge-radiora2#26]
All keypad buttons all showing up with same device name on iPhone - not individual button names
See attached files
![screenshot hoobs](https://user-images.githubusercontent.com/995235/204582448-0e23b5d3-1aa6-4709-ad95-50d2f4a77cdd.png)
[rr2 plugin config text.txt](https://github.com/senorshaun/homebridge-radiora2/files/10114869/rr2.plugin.config.text.txt)
[Hoob log](https://github.com/senorshaun/homebridge-radiora2/files/10114871/Hoob.log)
![image2](https://user-images.githubusercontent.com/995235/204582472-e5daa3db-28f3-45be-82ca-3ad02aaa2ab0.png)
![image1](https://user-images.githubusercontent.com/995235/204582482-4e27d08d-9179-4d95-aaa8-6af3f0dab97d.png)

**Comment by senorshaun** [comment]
Yeah it's super annoying. It seems that this is a change with iOS 16. I originally created a bug issue for it, issue #24 but it doesn't seem that there is a solution. Looking at other homebridge plugins, no one has an actual fix on the dev side.
The solution, on the user side, is to go in to the home app, delete the name of the accessory and you will see the actual button name in grey, as a placeholder. Type that in as the name.

**Comment by jfinestone** [comment]
Thank you.  This work around worked.  Can you keep this issue open in order
to update me/us when the issue has been resolved (likely by
Homebridge/Apple)?  Any idea what causes this.

**Comment by scott-hpe** [comment]
I was wondering about this too and thinking that is the _model_ or the _serial number_ field was unique, it might cause HomeKit populate the right name? Maybe exposure the field with the _button id_ value appended to it?

![IMG_0118](https://github.com/senorshaun/homebridge-radiora2/assets/95888238/397c34ee-fe2e-4ebf-87b3-5eb528dc228d)

**Comment by jfinestone** [comment]
Curious if you were ever able to solve this issue.  Also, the workaround of clicking the X to show the original named greyed-out no longer works.  Thank you.
